src/Compilers/CSharp/Portable/Symbols/Source/SourceUserDefinedOperatorSymbolBase.cs

```diff
@@ -133,6 +133,8 @@ protected override void MethodChecks(DiagnosticBag diagnostics)
                 allowThis: false,
                 diagnostics: diagnostics);
 
+            ParameterHelpers.EnsureIsReadOnlyAttributeExists(_lazyParameters, diagnostics, modifyCompilationForRefReadOnly: true);
+
```

Comment:
We don't parse ref return operators in source code.

---
In reply to: [115868321](https://github.com/dotnet/roslyn/pull/18715#discussion_r115868321) [](ancestors = 115868321)